tests/unit/index.html

```diff
@@ -21,8 +21,6 @@
         <div class="progress"></div>
     </div>
 
-    <script src="../bind.polyfill.js"></script>
```

Comment:
Removed in https://github.com/shower/core/commit/f065286f513f0fdc319183b7f7a61bbc2a7bf566.